ci-operator/step-registry/openshift/e2e/openstack/proxy/openshift-e2e-openstack-proxy-workflow.yaml

```diff
@@ -0,0 +1,25 @@
+workflow:
+  as: openshift-e2e-openstack-proxy
+  steps:
+    pre:
+      - chain: ipi-openstack-pre
+    test:
+      - ref: openshift-e2e-test
+    post:
+      - chain: ipi-openstack-post
+    env:
+      CONFIG_TYPE: "proxy"
+      BASE_DOMAIN: shiftstack.devcluster.openshift.com
+      TEST_SUITE: experimental/reliability/minimal
+      # These tests fail when we use HTTP_PROXY / HTTPS_PROXY to reach the OCP cluster and run tests against it.
+      # This will be investigated later, but in the meantime we skip them to be able to test the rest and
+      # provide good coverage.
+      TEST_SKIPS: >-
+        Kubectl Port forwarding With a server listening on 0.0.0.0 should support forwarding over websockets \|
+        Kubectl client Simple pod should support exec through kubectl proxy \|
+        Kubectl client Simple pod should support exec through an HTTP proxy \|
+        Kubectl Port forwarding With a server listening on localhost should support forwarding over websockets
```

Comment:
you'll need the ones from https://github.com/openshift/release/pull/21035/commits/902efde0556907f33a57df99d4a2e3972622f693 as well, I think.